src/main/java/com/ays/location/model/dto/request/UserLocationRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.ays.location.model.dto.request;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * A DTO class representing the request data for updating user location.
+ * <p>
+ * This class provides getters and setters for the latitude, and longitude fields.
+ * It also includes a builder pattern implementation for constructing instances of this class with optional parameters.
+ * <p>
+ * The purpose of this class is to encapsulate the request data related to updating user location, allowing for easy
+ * transfer of the data between different layers of the application.
+ */
+@Data
```

Comment:
Request objelerde `@Data` anotasyonuyla beraber gelen diğer anotasyonlara ihtiyacımız yok ise onları toplu bir şekilde eklemekten kaçınalım. Sadece ihtiyacımız olan anotasyonları ekleyelim. Request için normal şartlar altında `@Getter` ve `@Builder` anotasyonlarının yeterli olması gerekiyor.

Reply:
Gerekli düzeltmeyi yaptım. `UserAssignment `ve `Assignment `nasıl ikiye bölebiliriz.

Reply:
İleride 2 ayrı tablo olarak yani `AYS_USER_ASSIGNMENT` ve `AYS_ASSIGNMENT` tablolarını oluşturduğumuzda bu yapıya dönebiliriz.